### PR TITLE
add citations page and many other fixes

### PR DESCRIPTION
**Pull request by malachig**

### CODE REVIEW

File: docs/about/meetings.rst [hunk 1]
```diff
@@ -5,7 +5,25 @@ The CIViC team regularly holds curation jamborees and development hackathons. At
...
-:guilabel:`No upcoming events currently scheduled.`
+:guilabel:`Next planned event.`
+
+.. rubric:: 2022 CIViC Hackathon and Curation Jamboree
+
+----
+
+:Dates: Jul 30th, 2022
+:Location: Washington University in St. Louis
+:Sign Up: `Eventbrite: Cancer Variant Interpretation Hackathon and Curation Jamboree 2022 <https://eventbrite.com/e/cancer-variant-interpretation-hackathon-and-curation-jamboree-tickets-48287431006>`_
```
Comment (kkrysiak) [inline docs/about/meetings.rst:16]: This is the wrong link.

**Review by kkrysiak — APPROVED** [review]
Looks good.

File: docs/about/meetings.rst (hunk 1 above)
Comment (kkrysiak) [reply docs/about/meetings.rst:16]: Should be this one: https://www.eventbrite.com/e/cancer-variant-interpretation-hackathon-and-curation-jamboree-tickets-327320021997

Comment (malachig) [reply docs/about/meetings.rst:16]: Good catch @kkrysiak !

**Pull request merged**